doc/glossary.rst

```diff
@@ -14,186 +14,193 @@ reflect either technical terms or legacy ways of referring to Ceph systems.
-		CRUSH map. See `Cluster Map`_ for details.
-
-	Ceph Object Storage
-		The object storage "product", service or capabilities, which consists
-		essentially of a Ceph Storage Cluster and a Ceph Object Gateway.
-
-	Ceph Object Gateway
-	RADOS Gateway
-	RGW
-		The S3/Swift gateway component of Ceph.
-
-	Ceph Block Device
-	RBD
-		The block storage component of Ceph.
-
-	Ceph Block Storage
-		The block storage "product," service or capabilities when used in
-		conjunction with ``librbd``, a hypervisor such as QEMU or Xen, and a
-		hypervisor abstraction layer such as ``libvirt``.
-
+	Ceph Dashboard
 	Ceph File System
 	CephFS
 	Ceph FS
-		The POSIX filesystem components of Ceph. Refer
-		:ref:`CephFS Architecture <arch-cephfs>` and :ref:`ceph-file-system` for
-		more details.
+                The POSIX filesystem components of Ceph. Refer :ref:`CephFS
+                Architecture <arch-cephfs>` and :ref:`ceph-file-system` for
+                more details.
 
-	Cloud Platforms
-	Cloud Stacks
-		Third party cloud provisioning platforms such as OpenStack, CloudStack,
-		OpenNebula, Proxmox VE, etc.
+	Ceph Interim Release
+                Versions of Ceph that have not yet been put through quality
+                assurance testing, but may contain new features.
 
-	Object Storage Device
-	OSD
-		A physical or logical storage unit (*e.g.*, LUN).
-		Sometimes, Ceph users use the
-		term "OSD" to refer to :term:`Ceph OSD Daemon`, though the
-		proper term is "Ceph OSD".
+	Ceph Test Framework
+	Ceph Kernel Modules
+                The collection of kernel modules which can be used to interact
+                with the Ceph System (e.g., ``ceph.ko``, ``rbd.ko``).
 
-	Ceph OSD Daemon
-	Ceph OSD Daemons
+	Ceph Manager
+	Ceph Manager Dashboard
+	Ceph Metadata Server
+	Ceph Monitor
+	Ceph Node
+	Ceph Object Gateway
+	Ceph Object Storage
+                The object storage "product", service or capabilities, which
+                consists essentially of a Ceph Storage Cluster and a Ceph Object
+                Gateway.
+
+	Ceph Object Store
 	Ceph OSD
 		The Ceph OSD software, which interacts with a logical
 		disk (:term:`OSD`). Sometimes, Ceph users use the
 		term "OSD" to refer to "Ceph OSD Daemon", though the
 		proper term is "Ceph OSD".
 
-	OSD id
-		The integer that defines an OSD. It is generated by the monitors as part
-		of the creation of a new OSD.
+	Ceph OSD Daemon
+	Ceph OSD Daemons
+	Ceph Point Release
+		Any ad-hoc release that includes only bug or security fixes.
 
-	OSD fsid
-		This is a unique identifier used to further improve the uniqueness of an
-		OSD and it is found in the OSD path in a file called ``osd_fsid``. This
-		``fsid`` term is used interchangeably with ``uuid``
+	Ceph Project
+                The aggregate term for the people, software, mission and
+                infrastructure of Ceph.
 
-	OSD uuid
-		Just like the OSD fsid, this is the OSD unique identifier and is used
-		interchangeably with ``fsid``
+	Ceph Platform
```

Comment:
@zdover23 This appears to be in the wrong place.

Reply:
@colemitchell I've put it in the right place.